### PR TITLE
Adds Refund.md & changes PullPayments.md

### LINKED ISSUE #605: Create Refund.md and separate from PullPayment.md
Refunding should be a separate document nested under pull payments.

- [ ] Create Refund.md and explain the workflow 
- [ ] Remove Refund paragraph from pull payments but put a reference to it
- [ ] Edit structure and nest Refund's under Pull Payments

### PR DESCRIPTION
**Pull request by Zaxounette**
### PullPayments.md
* Transfers the `create refund` § from Pullpayments.md to Refund.md
* Fixes typos and adds spacing for clarity in PullPayments.md
* Adds sentence in PullPayments.md directing to Refund.md

### Refund.md

* Adds Refund.md - Used the § on the blog post featuring the v1.0.5.2
* Adds spacing between wording and images in the transferred §

### Config.js
* Nests Refund.md under PullPayments.md in config.js

Closes #605

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -222,6 +222,7 @@ module.exports = {
           ["/Wallet", "Wallet"],
           ["/Invoices", "Invoices"],
           ["/PullPayments", "Pull payments"],
```
Comment (britttttk) [inline docs/.vuepress/config.js:224]:
```suggestion
          ["/PullPayments", "Pull Payments"],
```

File: docs/Refund.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+, a merchant just needs to go to the invoice, click Issue refund , select the appropriate rate and share a link with a customer.
+
+The customer will then input their address and claim a refund. Once a customer claims a refund, the merchant will get a notification and can authorize it in a click.
+In future releases, we will look into payouts which are approved to be paid out instantly, without the authorization, in case a merchant prefers that.
+
+## Create a refund
+
+* After an invoice has been confirmed, go in the Invoices page and click `Details` on the invoice.
+
+![9](./img/pull-payments/9.png)
```
Comment (britttttk) [inline docs/Refund.md:17]: Can you update the moved image names ``

File: docs/.vuepress/config.js [hunk 3]
```diff
@@ -222,6 +222,7 @@ module.exports = {
...
+          ["/Refund", "Refunds"],
```
[the remark this answers is not in the document]
Comment (Zaxounette) [reply docs/.vuepress/config.js:225]: Oh, true that.
I might have misunderstood what @pavlenex meant by "nesting".

File: docs/Refund.md (hunk 2 above)
Comment (Zaxounette) [reply docs/Refund.md:17]: The image hasn't been moved, so the link is still valid.

Should I move the image and create a new "refund" folder in /IMG ?

File: docs/.vuepress/config.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (pavlenex) [reply docs/.vuepress/config.js:225]: Yes, I meant refunds as part of Pull payments.
Pull payments (parent), Refund (child).

[the remark this answers is not in the document]
Comment (Zaxounette) [reply docs/.vuepress/config.js:225]: Alright, fixing now =)

**Comment by Zaxounette** [comment]
Check failing on the build.
With my luck it's just a comma I can't find that's missing.

**Comment by pavlenex** [comment]
@Zaxounette Take a look at how others are nested, this won't build for me at all.

**Comment by Zaxounette** [comment]
> @Zaxounette Take a look at how others are nested, this won't build for me at all.

That's what I did.
I'll check again !

File: docs/Refund.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Refunds
+
+Refunds are one of the use cases that use the `Pull Payments` feature.
+
+Up until now, merchants had to manually issue refunds.
+The process required a lot of back and forth between a sender and a receiver. The merchant had to ask for a refund address, calculate the preferred rate, manually add that to the wallet and broadcast the transaction.
+
+, a merchant just needs to go to the invoice, click Issue refund , select the appropriate rate and share a link with a customer.
```
Comment (pavlenex) [inline docs/Refund.md:8]: Typo

File: docs/Refund.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Refunds
+
+Refunds are one of the use cases that use the `Pull Payments` feature.
```
Comment (pavlenex) [inline docs/Refund.md:3]: imo it's better to say that refunds are built on top of the pull payments feature

File: docs/Refund.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+Up until now, merchants had to manually issue refunds.
+The process required a lot of back and forth between a sender and a receiver. The merchant had to ask for a refund address, calculate the preferred rate, manually add that to the wallet and broadcast the transaction.
+
+, a merchant just needs to go to the invoice, click Issue refund , select the appropriate rate and share a link with a customer.
+
+The customer will then input their address and claim a refund. Once a customer claims a refund, the merchant will get a notification and can authorize it in a click.
+In future releases, we will look into payouts which are approved to be paid out instantly, without the authorization, in case a merchant prefers that.
```
Comment (pavlenex) [inline docs/Refund.md:11]: I'd make this more formal. The future versions of btcpay server may.... sth like that.

File: docs/Refund.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+* Click `Issue a refund`
+
+![10](./img/pull-payments/10.png)
+
+* Select the way you want to issue the refund
+
+![11](./img/pull-payments/11.png)
+
+* You can now share the pull payment link with the customer
```
Comment (pavlenex) [inline docs/Refund.md:27]: > You can now 

I'd remove this

**Comment by pavlenex** [comment]
@Zaxounette  Works now a few minor notes ^.

File: docs/Refund.md (hunk 4 above)
Comment (Zaxounette) [reply docs/Refund.md:8]:
```suggestion
A merchant just needs to go to the invoice, click Issue refund , select the appropriate rate and share a link with a customer.
```

File: docs/Refund.md (hunk 5 above)
Comment (Zaxounette) [reply docs/Refund.md:3]:
```suggestion
Refunds are one of the applications built on top of the `Pull Payments` feature.
```

File: docs/Refund.md (hunk 7 above)
Comment (Zaxounette) [reply docs/Refund.md:27]:
```suggestion
* Share the pull payment link with the customer
```

File: docs/Refund.md (hunk 6 above)
Comment (Zaxounette) [reply docs/Refund.md:11]: > The future versions of btcpay server may.... sth like that.

Pardon ?

Comment (Zaxounette) [reply docs/Refund.md:11]: Ah sorry, I understand what you were saying now !

Comment (Zaxounette) [reply docs/Refund.md:11]:
```suggestion
Future versions of BTCPay Server will enable instant payouts as an option, without the need for merchant approval.
```

File: docs/Refund.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+# Refunds
+
+Refunds are one of the applications built on top of the `Pull Payments` feature.
```
Comment (pavlenex) [inline docs/Refund.md:3]: Can we link to pull payments instead? It's good to have links from one page to another, sorry , misted this one earlier.

Comment (Zaxounette) [reply docs/Refund.md:3]: No problem ! 
```suggestion
Refunds are one of the applications built on top of the [Pull Payments](./PullPayments.md) feature.
```

**Pull request merged**